src/content/docs/setup/configuration/aem-assets-configuration.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: AEM Assets integration
+description: Learn how to connect your Edge Delivery Services storefront to your AEM Assets integration.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import Vocabulary from '@components/Vocabulary.astro';
+import { Tabs, TabItem, Aside } from '@astrojs/starlight/components';
+
+The AEM Assets integration allows your storefront to display product images that are managed in AEM Assets instead of traditional Commerce-hosted images. This integration provides enhanced image management capabilities, including advanced optimization, cropping, and delivery through Adobe's Content Delivery Network (CDN).
+
+## Boilerplate update required
+
+Only one update is required: set `"commerce-assets-enabled": true` in your `config.json` file, renamed from `/demo-config.json`. See [Storefront configuration](/setup/configuration/commerce-configuration/) for the full configuration model.
+
+```json title="config.json" ins={'+':4}
+{
+  "public": {
+    "default": {
+      "commerce-assets-enabled": true
+    }
+  }
+}
+```
+
+The Commerce drop-ins automatically detect the `commerce-assets-enabled` configuration and adjust image handling accordingly. See the [demo-config.json](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/demo-config.json) file in the boilerplate repository. 
```

Comment:
Do not link to this file. Reference the live configuration on aemshop.net at most. https://www.aemshop.net/config.json
```suggestion
The Commerce drop-ins automatically detect the `commerce-assets-enabled` configuration and adjust image handling accordingly. See the [demo configratuion](https://www.aemshop.net/config.json). 
```